### PR TITLE
added textfiles folder and documents to Users/abehenderson

### LINKED ISSUE #22: New directories needed in Users/abehenderson
These sub-issues need to be resolved:

Using the command line only, please create the following directories and/or files in the directory /Users/abehenderson:

- [x] /textfiles/log.txt, with a small amount of content (your choice)
- [x] /textfiles/subfiles/log.md, with a small amount of content written in markdown syntax (perhaps a list, with some headings, bold and emphasized text

Proper steps to solve this issue are: 
1. make sure you have an up to date copy of main locally by running `git checkout main`, `git pull origin main` then checkout a new feature branch with `git checkout -b [name of your feature branch goes here]`
2. make the changes in your feature branch, then create a new PR like you've done previously
3. the top line of your PR comment should reference this issue using the format here: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
4. Slack me with questions

### PR DESCRIPTION
**Pull request by a-hend**
Closes #22 

Created textfiles folder in Users/abehenderson.
Created log.txt in the textfiles folder and added some text.
Created subfiles folder in Users/abehenderson/textfiles.
Created log.md in the subfiles folder and made some notes for Friday Night Fish Tacos in markdown.

### CODE REVIEW

File: Users/abehenderson/textfiles/log.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+I used mkdir Users/abehenderson/textfiles to create a new directory 'textfiles'.
```
Comment (C-STYR) [inline Users/abehenderson/textfiles/log.txt:1]: Let's make sure to use backticks when we write terminal commands, here and below.

File: Users/abehenderson/textfiles/log.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+I used touch Users/abehenderson/textfiles/log.txt to create this file 'log.txt'
+I could not find a terminal command to open log.txt in the editor window, so I clicked on it with my mouse.  
+Did I cheat?
```
Comment (C-STYR) [inline Users/abehenderson/textfiles/log.txt:4]: Yes! Look for a new issue ...

**Review by C-STYR — CHANGES_REQUESTED** [review]

File: Users/abehenderson/textfiles/log.txt (hunk 1 above)
Comment (a-hend) [reply Users/abehenderson/textfiles/log.txt:1]: Sooo...
I used `mkdir Users/abehenderson/textfiles` to create a new directory 'textfiles'.

...?

File: Users/abehenderson/textfiles/log.txt (hunk 2 above)
Comment (a-hend) [reply Users/abehenderson/textfiles/log.txt:4]: Ok, so is issue #27 Opening files in Terminal linked to this issue now?

Comment (a-hend) [reply Users/abehenderson/textfiles/log.txt:4]: or is it separate?

File: Users/abehenderson/textfiles/log.txt (hunk 1 above)
Comment (C-STYR) [reply Users/abehenderson/textfiles/log.txt:1]: Yes

File: Users/abehenderson/textfiles/log.txt (hunk 2 above)
Comment (C-STYR) [reply Users/abehenderson/textfiles/log.txt:4]: Not linked, just a new issue dealing specifically with what you brought up on line 3.

**Review by C-STYR — DISMISSED** [review]

**Review by C-STYR — COMMENTED** [review]
Screen Shot 2022-10-09 at 11 11 54 AM
Empty file! You need to make sure it's saved, then add and commit again.

**Comment by a-hend** [comment]
I keep thinking that committing is saving....argh.  But I need to save on my device first, then commit to branch, then push, then pull request?  I think I'm skipping the first step.  I'll try again.

**Comment by a-hend** [comment]
Do I need to do something with this one before it can be reviewed again and merged?

**Review by C-STYR — APPROVED** [review]

**Pull request merged**